### ISSUE TITLE
Bad indentation in DataView's example ColumnDataModel

### ISSUE THREAD

**Issue by xamcost** [enthought/pyface#631]
Hi everyone! It seems that right now, you cannot run `pyface.examples.data_view.column_example`: it will crash due to an `IndentationError`:

```
Traceback (most recent call last):
  [...]
  File "/.../data_view/column_data_model.py", line 213
    row_info.set_value(obj, value)
                                 ^
IndentationError: unindent does not match any outer indentation level
```

The problem lies in `pyface.examples.data_view.column_data_model`: 

https://github.com/enthought/pyface/blob/990cbd209e4ba0c172978c797d856da3c86a4736/examples/data_view/column_data_model.py#L217

where a space is missing to get a correct indentation. Putting that extra space solves the issue.

On a lighter note, in that same file, there is a missing import: `DataViewSetError`.

**Comment by kitchoi** [comment]
Hi @xamcost, thank you for the report!
Yes this is being fixed as part of #574

The examples were only added for review purposes, there are still a lot rough corners that need to be smoothed out before the release of this new feature. IOW, it is not ready for public consumption yet, please refrain from adapting it in anything production like. Experimentation is very welcome though!
(Related: https://github.com/enthought/pyface/issues/569)

**Comment by corranwebster** [comment]
Resolved in #574
